app/main.py

```
@@ -15,6 +19,100 @@ def check_email(payload: EmailIn):
     out = score_email(payload.text)
     return {"channel": "email", **out}
 
+@app.post("/check_eml")
+async def check_eml(file: UploadFile = File(...)):
```

Ruff (0.12.2)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)